microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -56,6 +120,7 @@ The following table lists the currently available permission strings to use in y
 | `debugger` | Gives your extension access to the `chrome.debugger` API. |
 | `declarativeContent` | Gives your extension access to the `chrome.declarativeContent` API. |
 | `declarativeNetRequest` | Gives your extension access to the `chrome.declarativeNetRequest` API. |
+| `declarativeNetRequestWithHostAccess` | `declarativeNetRequest` API and `declarativeNetRequestWithHostAccess`both allow the extension to use the `declarativeNetRequest` API. The `declarativeNetRequest` permission allows the extension to block and upgrade requests without host permissions but needs host permissions to redirect requests or modify headers. The `declarativeNetRequestWithHostAccess` permission always needs host permissions to the request URL and the initiator to act on the request. |
```

Comment:
A space is needed after `declarativeNetRequestWithHostAccess` in the line:
`declarativeNetRequest` API and `declarativeNetRequestWithHostAccess`both allow the extension to use

Reply:
added space